json-rpc/src/views.rs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use hex;
+use libra_types::{account_config::AccountResource, byte_array::ByteArray};
+use serde::Serialize;
+
+#[derive(Serialize)]
+pub struct AccountView {
```

Comment:
With AccountResource, LibraAccountResource (defined in libra-dev) and AccountView -- all three of them are basically same versions of each other. Not sure if there's any way to streamline them because they all have different use cases, but seems messy with multiple definitions of the same thing..

Reply:
`AccountResource` is main struct (original definition) used everywhere inside of Node code
`AccountView` is just view of it - representation for final JSON serialization. Scope is very clearly isolated to JSON RPC service 
`LibraAccountResource` is CFFI (core lib) specific. IMO eventually should be separated into different repo